src/context.rs

```diff
@@ -622,7 +622,7 @@ impl Context {
 
     pub fn get_font_face(&self) -> FontFace {
         unsafe {
-            FontFace(ffi::cairo_get_font_face(self.0))
+            from_glib_none(ffi::cairo_get_font_face(self.0))
```

Comment:
Btw, was this a bug? Since previously, `cairo_font_face_reference` was not called after calling `cairo_get_font_face`. So when `FontFace` gets dropped, `cairo_font_face_destroy` will get invoked possibly deallocating the original font_face while still in use.

Or am I doing [it](https://github.com/johncf/cairo/blob/d65b2531/src/fonts.rs#L269) wrong?